### PR TITLE
badges: style snippets

### LINKED ISSUE #72: semantic-ui badges do not display correctly
The doi badges do not display the content in a "snippet" as before. Some even go out of the bounds of the modal.

Screenshot 2020-05-29 at 14 30 25

### PR DESCRIPTION
**Pull request by KonstantinaStoikou**
Closes #72 

Screenshot:
Screenshot 2020-12-02 at 11 06 55

### CODE REVIEW

**Review by ppanero — COMMENTED** [review]
I think the problem here is two folded:

1- The module lacks the bootstrap folder (where the changed file should be). And should work (:tm:) in bootstrap since the `code` tag is [defined there](https://getbootstrap.com/docs/4.0/content/code/). Should the folder be created? @lnielsen 

2- The file to change is actually the semant-ui one: https://github.com/inveniosoftware/invenio-formatter/tree/master/invenio_formatter/templates/semantic-ui/invenio_formatter/macros . At a first glance Semantic-UI does not have it, but I think it can be achieved playing with the [`segment`](https://semantic-ui.com/elements/segment.html) class.

File: invenio_formatter/templates/semantic-ui/invenio_formatter/macros/badges.html [hunk 1]
```diff
@@ -9,37 +9,44 @@
 
 {# Generate a list different ways to embed a badge. #}
 {%- macro badges_formats_list(image_filename, target, alternative='DOI') %}
+
+  <style>
+    .ui.message {
+      overflow: auto;
+    }
+  </style>
```
Comment (ppanero) [inline invenio_formatter/templates/semantic-ui/invenio_formatter/macros/badges.html:17]: Is this needed?

**Review by ppanero — COMMENTED** [review]
LGTM, Just removal of the unused style HTML tag. Also the screenshot is updated?

File: invenio_formatter/templates/semantic-ui/invenio_formatter/macros/badges.html (hunk 1 above)
Comment (KonstantinaStoikou) [reply invenio_formatter/templates/semantic-ui/invenio_formatter/macros/badges.html:17]: Yes, because before it got out of the modal bounds.

**Review by ppanero — APPROVED** [review]

File: invenio_formatter/templates/semantic-ui/invenio_formatter/macros/badges.html (hunk 1 above)
Comment (ppanero) [reply invenio_formatter/templates/semantic-ui/invenio_formatter/macros/badges.html:17]: True, sorry, I missed that

File: invenio_formatter/templates/semantic-ui/invenio_formatter/macros/badges.html [hunk 2]
```diff
@@ -9,37 +9,44 @@
 
 {# Generate a list different ways to embed a badge. #}
 {%- macro badges_formats_list(image_filename, target, alternative='DOI') %}
+
+  <style>
```
Comment (lnielsen) [inline invenio_formatter/templates/semantic-ui/invenio_formatter/macros/badges.html:13]: Minor: We should not add inline styles in the pages. I will need to check where to add it exactly, so no action needed on your side right now.

Comment (lnielsen) [reply invenio_formatter/templates/semantic-ui/invenio_formatter/macros/badges.html:13]: I think this code has to go to Invenio-Theme on:

https://github.com/inveniosoftware/invenio-theme/blob/master/invenio_theme/assets/semantic-ui/less/invenio_theme/theme/collections/message.overrides

**Pull request merged**